docs/Fields.md

```diff
@@ -61,6 +61,16 @@ Then you can display the author first name as follows:
 
 **Tip**: If your interface has to support multiple languages, don't use the `label` prop, and put the localized labels in a dictionary instead. See the [Translation documentation](./Translation.html#translating-resource-and-field-names) for details.
 
+## `<AddressField>`
+
+Displays an address as addresse link with the geo or Apple maps URI scheme
```

Comment:
Displays an address location as a clickable link. On a mobile phone, it will open the map application (plan for IOS, google maps, waze or whatever for Android), center on the location. On a desktop, it has almost no effect for now.